src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -758,17 +759,17 @@ protected override bool SetResultCore(PhysicalConnection connection, Message mes
                                     server.Multiplexer.Trace("Auto-configured databases: " + dbCount);
                                     server.Databases = dbCount;
                                 }
-                                else if (key.IsEqual(CommonReplies.slave_read_only))
+                                else if (key.IsEqual(CommonReplies.slave_read_only) || key.IsEqual(CommonReplies.replica_read_only))
                                 {
                                     if (val.IsEqual(CommonReplies.yes))
                                     {
-                                        server.SlaveReadOnly = true;
-                                        server.Multiplexer.Trace("Auto-configured slave-read-only: true");
+                                        server.ReplicaReadOnly = true;
+                                        server.Multiplexer.Trace("Auto-configured replica-read-only: true");
                                     }
                                     else if (val.IsEqual(CommonReplies.no))
                                     {
-                                        server.SlaveReadOnly = false;
-                                        server.Multiplexer.Trace("Auto-configured slave-read-only: false");
+                                        server.ReplicaReadOnly = false;
+                                        server.Multiplexer.Trace("Auto-configured replica-read-only: false");
```

Comment:
I'm good with this, just noting it's not technically accurate as we could be configuring a pre-5 server and this isn't the config value - don't care, log says what we need?